src/core/tritonserver.cc

```diff
@@ -552,7 +552,7 @@ TRITONSERVER_LogIsEnabled(TRITONSERVER_LogLevel level)
   return false;
 }
 
-TRITONSERVER_Error*
+void
```

Comment:
I can see why you want to do this. But I would instead just like to make it easier to ignore the error. So can you add IGNORE_ERROR to backend_utils.h and use it in LOG_IF_ERROR. Also add a LOG macro that uses LOG_IF_ERROR with success error so that it always logs and in the backends replace TRITONSERVER_LogMessage with the LOG macro.